scripts/release/create_rc.mjs

```diff
@@ -102,8 +102,8 @@ if (!(await confirm(`Do you want to create a release candidate for ${versionMMP}
 // Create a new release branch `rc/v1.1.0` at commit `9fceb02`
 shell(`git checkout -b ${rcBranchName} ${commit}`);
 
-// Set monorepo version to `1.1.0`
-shell(`lerna version ${packageVersion} --no-git-tag-version --force-publish --yes`);
+// Set monorepo version to `1.1.0`. Use tilde to apply only patch releases, not minor
+shell(`node scripts/release/set_version.mjs ${packageVersion} tilde`);
```

Comment:
tilde here means inter-monorepo deps with be `~0.40.0` such that only patch releases can be installed in that range. `^0.40.0` applies to minor releases too.